### ISSUE TITLE
Header overlaps on smaller screens

### ISSUE THREAD

**Issue by ajdapretnar** [biolab/orange-hugo#167]
Picture is worth a thousand words.

![Screen Shot 2020-08-04 at 11 02 59](https://user-images.githubusercontent.com/12524972/89275578-bde04a80-d642-11ea-9fce-cfcb72c1aca5.png)

**Comment by markotoplak** [comment]
Also, we have too many option. I recommend removing "Screenshots".

**Comment by markotoplak** [comment]
And we should probably remove some more. I doubt that having "Workshops" and "Donate" in the top menu (both are not the first thing a webpage user wants) gain anything, except that they make options more confusing.

**Comment by ajdapretnar** [comment]
Screenshots is our fifth most frequent page on site. It has over 1000 views each week. I also don't like it, but it seems to be popular.

**Comment by markotoplak** [comment]
Oh, and we can remove the blog, because there are stories just under the header.

**Comment by ajdapretnar** [comment]
I would say blog and features. Features just scrolls further down the site.

**Comment by markotoplak** [comment]
@ajdapretnar, yes, true, we should be careful about screenshots. But if they were not there, perhaps users would just click features? Or vice versa (which you suggest).

**Comment by markotoplak** [comment]
So blog + features + workshops and we are happy? :D

**Comment by ajdapretnar** [comment]
Why workshops? 😕 It is a way of promoting workshops that we offer.

**Comment by ajdapretnar** [comment]
I think @BlazZupan has to have a say in this.

**Comment by markotoplak** [comment]
@ajdapretnar, I do not know if that is a good place for promoting workshops. It does make them visible, but decreases website usability. I do not know what is a good compromise, but on the Blog and Features we agree, right?

**Comment by ajdapretnar** [comment]
- Blog goes to blog banner, rename to All posts.
- Remove Features.
- Move Screenshots to first two feature desc.
